src/main/resources/standardTestCases/globbingindex/globbingindex.wdl

```diff
@@ -0,0 +1,60 @@
+task glob_task {
+    command {
+        echo "staticFile" > staticFile.txt
+        echo "staticArray" > staticArray.txt
+        echo "globFile" > globFile.txt
+        echo "globArray" > globArray.txt
+    }
+
+    output {
+        Array[File] staticFiles = ["staticFile.txt"]
+        Array[File] staticRef = staticFiles
+        File staticFile = staticFiles[0]
+        Array[File] staticArray = ["staticArray.txt"]
+        File globFile = glob("globFile.txt")[0]
+        Array[File] globArray = glob("globArray.txt")
```

Comment:
ToL: Bonus points for `File globFile2 = glob("globFile.txt")[read_int("intFile.txt")]`...